src/pfe/portal/modules/User.js

```diff
@@ -962,13 +924,9 @@ module.exports = class User {
       return retval;
```

Comment:
Again, this can just return the list. `DELETE /api/v1/registrysecrets` knows that's a 200.